e-moe/guzzle6-bundle/1.1/etc/packages/emoe_guzzle_bundle.yaml

```diff
@@ -0,0 +1,4 @@
+emoe_guzzle:
+    log:
+        enabled: true # Logging requests to Monolog
+        format: 'Guzzle: [{ts}] "{method} {uri} HTTP/{version}" {code}' # Optional log format customization
```

Comment:
does the bundle actually require to provide a log format ? If it has a format making sense, it should be the default one